package.json

```diff
@@ -43,9 +43,9 @@
     "test:integration": "npm run mocha -- 'test/integration/**/*.test.ts'",
     "test:compute": "npm run mocha -- 'test/integration/ComputeFlow.test.ts'",
     "test:integration:cover": "nyc --report-dir coverage/integration --exclude 'src/@types/**/*' --no-clean npm run test:integration",
-    "create:guide": "chmod +x ./scripts/createCodeExamples.sh && ./scripts/createCodeExamples.sh",
-    "create:guide:mac": "chmod +x ./scripts/createCodeExamples-mac.sh && ./scripts/createCodeExamples-mac.sh",
-    "commit:guide": "chmod +x scripts/commitChanges.sh && scripts/commitChanges.sh"
+    "create:guide": "chmod +x ./scripts/createCodeExamples.sh && ./scripts/createCodeExamples.sh test/integration/CodeExamples.test.ts",
```

Comment:
doing `chmod +x` every time this runs seems unnecessary and I doubt this works on Windows. Do this locally once, then commit the file as file attributes are version controlled so in end you commit the executable script